app/src/editor/canvas/components/Toolbar.pcss

```diff
@@ -352,3 +354,11 @@
   left: 50%;
   transform: translate(-50%, 0);
 }
+
+.DashboardEntityName {
+  margin-left: 40px;
```

Comment:
Chatted with @mattatgit about this and this line can be removed so the left and right margins are symmetric.